superset/db_engine_specs/base.py

```diff
@@ -941,7 +975,8 @@ def execute(cls, cursor: Any, query: str, **kwargs: Any) -> None:
         try:
             cursor.execute(query)
         except Exception as ex:
-            raise cls.get_dbapi_mapped_exception(ex)
+            dbapi = importlib.import_module(cursor.__module__)
```

Comment:
I'm not super happy with this, since in theory the cursor could be defined in a submodule that doesn't have the exceptions in scope.

The alternative is to pass the database or engine instance to `db_engine_spec.execute` and `db_engine_spec.fetch_data`.

Reply:
When forcing an error on `sqlite`, I got the following exception on this line:
![image](https://user-images.githubusercontent.com/33317356/108720682-c9855700-7529-11eb-90b9-88cd25497612.png)